applications/luci-app-travelmate/po/ja/travelmate.po

```diff
@@ -123,13 +123,13 @@ msgstr ""
 "確認してください。"
 
 msgid "Force CCMP (AES)"
-msgstr ""
+msgstr "CCMP (AES)"
```

Comment:
You ignored the "Force" part, is this intentional? I would have expected something with 強制 or so.

Reply:
In contrast to "Automatic", I thought that meaning would be transmitted sufficiently to users without "Force". Also, because there is no "Force" in LuCI's dropdown list of the cipher setting in Wireless page , so there is also a purpose of unifying it.

Reply:
Makes sense (I was mostly curious, so it was not intended as an actual review).